README.md

```diff
@@ -3400,6 +3400,48 @@ while (cond){
 }
 ```
 
+
+### requireNewlineAfterArrayElements
+
+Requires each element in array on a single line.
+
+Type: `Boolean` or `Number` (maximum) or `Object` (maximum: Infinity, ignoreBrackets: false)
+
+Values: `true`
+
+Default: Infinity
+
+#### Example
+
+```js
+"requireNewlineAfterArrayElement": 3
```

Comment:
For the description of the rule i'm not sure what id does, could add more clarify it more explicitly?